packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -0,0 +1,146 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import {
+  createVar,
+  fallbackVar,
+  style,
+  styleVariants,
+} from '@vanilla-extract/css';
+
+const contrastColor = createVar();
+
+export const containerClass = style([
+  sprinkles({
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    height: 'min-content',
+    position: 'relative',
+  }),
+  {
+    border: `1px solid ${fallbackVar(contrastColor, vars.colors.$neutral6)}`,
+    borderLeftWidth: vars.sizes.$1,
+  },
+]);
+
+export const footerClass = style([
+  sprinkles({
+    marginY: '$md',
+  }),
+]);
+
+export const expandVariants = styleVariants(
+  {
+    true: true,
+    false: false,
+  },
+  (expand) => {
+    return [
+      containerClass,
```

Comment:
You only need to add this class if it isn't already being applied, but in this case it is getting applied twice because it is in both variants. How about we come up with a convention that we don't include them in variants moving forward is it is very clear what classes we are applying to elements in the jsx?

Reply:
I agree, it makes more sense to apply the `containerClass` directly into the component, when used in multiple variants